sqlscripts/replication/0002-use-jaeger.sql

```diff
@@ -0,0 +1 @@
+USE jaeger;
```

Comment:
is this really needed? the DB is specified in the connection string.

Reply:
If database jaeger doesn't exist, we won't be able to connect.

Reply:
but we use database in SQL connection string.

Reply:
https://github.com/go-sql-driver/mysql/issues/173#issuecomment-28539448